### DISCUSSION TITLE
IrScrutinizer leaves newcomer in the dark

### DISCUSSION THREAD

**Discussion by Dan-Essin** [bengtmartensson/IrScrutinizer#488] [Q&A]
This looks like an impressive app but...
I'm new here. In the hardware options Audio Port is the default. There is also IrWidget and some others I can't even speculate about.

What I've got in an Arduino Uno and as TSOP4838.

The docs are completely lack any description of how to actually connect something that could acquire a signal.
How would I use the Audio Port?
Where does IrWidget come from? Is there some way to implement it on Arduino? If so, how would I tell IrScrutinizer where/how to find the Arduino (e.g. COM6)?
Is there some other, intuitively obvious way to capture raw signal with Arduino and somehow import the into IrScrutinizer in some horrid, multi-step process for each one of the 9 buttons x 2 on my Mistubishi A/C remotes (each one may have its own protocol since they don't inter-operate).

Without a clue as to how to acquire a signal to scrutinize, the app is quite useless to me no matter how elegant it is.
Not trying to be facetious but is this somehow obvious to everyone but me? I confess that I flunked mind-reading 101 in college?

Please help!
Thank you

**Comment by bengtmartensson** [comment]
These questions are all answered in the [documentation](http://www.harctoolbox.org/irscrutinizer.html). It appears that you have not read it. There is also a [tutorial](http://www.hifi-remote.com/wiki/index.php?title=IrScrutinizer_Guide).

**Comment by henryhenson80** [comment]
https://www.harctoolbox.org/arduino_nano.html
